Doc/whatsnew/3.14.rst

```diff
@@ -180,24 +181,32 @@ asyncio
   reduces memory usage.
   (Contributed by Kumar Aditya in :gh:`107803`.)
 
-
-
 Deprecated
 ==========
 
-* Passing a complex number as the *real* or *imag* argument in the
+* :mod:`builtins`:
+  Passing a complex number as the *real* or *imag* argument in the
   :func:`complex` constructor is now deprecated; it should only be passed
   as a single positional argument.
   (Contributed by Serhiy Storchaka in :gh:`109218`.)
 
-* :term:`Soft deprecate <soft deprecated>` :func:`os.popen` and
+* :mod:`os`:
+  :term:`Soft deprecate <soft deprecated>` :func:`os.popen` and
   :func:`os.spawn* <os.spawnl>` functions. They should no longer be used to
   write new code.  The :mod:`subprocess` module is recommended instead.
   (Contributed by Victor Stinner in :gh:`120743`.)
 
-* Deprecate :meth:`symtable.Class.get_methods` due to the lack of interest.
+* :mod:`symtable`:
+  Deprecate :meth:`symtable.Class.get_methods` due to the lack of interest.
```

Comment:
Why aren't these deprecations included?

Reply:
Yep, will add them.

1. So the first one https://github.com/python/cpython/issues/109218 / https://github.com/python/cpython/pull/119620/files has no planned removal date, so will go into the future list.

2. The second https://github.com/python/cpython/issues/120743 / https://github.com/python/cpython/pull/120744 is a "soft deprecation" meaning it "does not imply scheduling the removal of the deprecated API". Hmm, so does this mean it does not belong in the "will be removed in the future" list?

3. The last one https://github.com/python/cpython/issues/119698 / https://github.com/python/cpython/pull/121902 will be removed in 3.16, so that's clear.

Reply:
Added!